Makefile.docker

```diff
@@ -2,6 +2,7 @@
 # SPDX-License-Identifier: Apache-2.0
 
 UTC_DATE=$(shell date -u "+%Y-%m-%d")
+REPOSITORY ?= quay.io
```

Comment:
nit: contrib/scripts/push_manifest.sh script is using term `DOCKER_REGISTRY` instead, so just small nitpick for consistency.

Reply:
good point, will switch.